source/reference/api/commit.txt

```diff
@@ -16,7 +16,11 @@ Description
 -----------
 
 Finalizes the sync between the source cluster and the destination
-cluster and stops continuous sync between clusters. ``commit`` also
+cluster and stops continuous sync between clusters. 
+
+.. _c2c-commit-alters:
```

Comment:
[suggestion] Optionally, I think we could maybe move this content that's under `c2c-commit-alters` to it's own "Commit Alters" section under Behavior lower down on the page. What do you think?

Reply:
I like this idea!